packages/create-react-admin/templates/local-auth-provider/authProvider.ts

```diff
@@ -0,0 +1,38 @@
+import { AuthProvider, HttpError } from 'react-admin';
+import data from './users.json';
+
+export const authProvider: AuthProvider = {
```

Comment:
You should add a comment explaining that this should not be used in production

Reply:
Done